src/content/developers/docs/smart-contracts/source-code-verification/index.md

```diff
@@ -0,0 +1,99 @@
+
+For a smart contract to be trustless, the contract code should be available for independent verification.  While the compiled bytecode for every smart contract is publicly available on the blockchain, low-level language is difficult to understand—for both developers and users. 
+
+Projects reduce trust assumptions by publishing the source code of their contracts. But this leads to another problem: it is difficult to verify that the published source code match the contract bytecode. In this scenario, the value of trustlessness is lost because users have to trust developers not to change a contract's business logic (i.e., by changing the bytecode) before deploying it on the blockchain. 
+
+Source code verification tools provide guarantees that a smart contract’s source code files matches the assembly code. The result is a trustless ecosystem, where users don’t blindly trust third parties and instead verify code before depositing funds into a contract. 
+
+### User Safety {#user-safety}
+
+With smart contracts, there’s usually a lot of money at stake. This calls for higher security guarantees and verification of a smart contract’s logic before using it. The problem is that unscruplous developers can deceive users by inserting malicious code in a smart contract. Without verification, malicious smart contracts can have [backdoors](https://www.trustnodes.com/2018/11/10/concerns-rise-over-backdoored-smart-contracts), controversial access control mechanisms, exploitable vulnerabilities, and other things that jeopardize user safety that would go undetected. 
+
+Publishing a smart contract's source code files makes it easier for those interested, such as auditors, to assess the contract for potential attack vectors. With multiple parties independently verifying a smart contract, users have stronger guarantees of its security. 
+
+## How to verify source code for Ethereum smart contracts {#source-code-verification-for-ethereum-smart-contracts}
+
+[Deploying a smart contract on Ethereum](/developers/docs/smart-contracts/deploying/) requires sending a transaction with a data payload (compiled bytecode) to a special address. The data payload is generated by compiling the source code, plus the [constructor arguments](https://docs.soliditylang.org/en/v0.8.14/contracts.html#constructor) of the contract instance appended to the data payload in the transaction. Compilation is deterministic, meaning it always produces the same output (i.e., contract bytecode) if the same source files, and compilation settings (e.g. compiler version, optimizer) are used. 
+
+![](https://github.com/emmanuel-awosika/ethereum-org-website/blob/0df62bc5ef7420929f079eb3174f1d69e5a2e3d3/src/content/developers/docs/smart-contracts/source-code-verification/Source%20code%20verification.png)
```

Comment:
I think there needs to be a relative link here. Check Accounts page for the example: https://github.com/ethereum/ethereum-org-website/blob/dev/src/content/developers/docs/accounts/index.md

Reply:
Also adding a higher resolution image: 
![source-code-verification](https://user-images.githubusercontent.com/13069972/178468962-d6cfe435-7e0c-4dd1-9242-817eee5ba2cb.png)